Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -98,8 +98,16 @@ - (nullable UIActivityViewController *)sharingActivityViewControllerWithTextSnip
     if (url) {
         [items addObject:[[NSURL alloc] initWithString:[NSString stringWithFormat:@"%@?%@", url.absoluteString, @"wprov=sfsi1"]]];
     }
+    
+    if (CLLocationCoordinate2DIsValid(self.coordinate)) {
```

Comment:
Ah good point. Originally it was all coordinate-based. Then I switched it to be MapItem based and didn't change this condition